source/support.html.erb

```diff
@@ -126,7 +126,7 @@ title: Support
           tom.dolan@digital.cabinet-office-gov.uk
         </a>
         <br />
-        Jess O’Leary, product manager -
+        Jess O'Leary, product manager -
```

Comment:
@jollery you could alternatively remove the right hand quote marks from your email address entirely. As I just tested, gmail doesn't care if they're there or not and delivers them anyway. It depends if you mind or not.

I'm sure @quis is correct about them being the right quote mark in written language.